src/router/legacyPermalinkManagement.routerPlugin.js

```diff
@@ -111,8 +111,16 @@ const handleLegacyParam = (
             // see adr/2021_03_16_url_param_structure.md
             break
 
-        // if no special work to do, we just copy past legacy params to the new viewer
+        case '3d':
+            // if the 3d flag is given without being placed behind the hash (meaning at startup),
+            // we need to make sure the projection is set to Mercator, otherwise the startup sequence
+            // ends up not working properly and center the view wrongly
+            if ((typeof legacyValue === 'string' && legacyValue === 'true') || legacyValue) {
+                newQuery['sr'] = WEBMERCATOR.epsgNumber
+            }
+        // no break so we go into default too
```

Comment:
`3d` is not a legacy parameter and we should never have it before the `#` of I'm not misteaken ?

Reply:
In some of our tests, it is.
And even though it's not legacy, I think we should make sure (advanced) users don't have to write this # to make it work.

We could think about having a dedicated file for current URL params than need special care, that would make more sense

Reply:
As discussed we do this later on